src/components/Button/Button.tsx

```diff
@@ -2,12 +2,20 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
 import * as React from 'react';
-import { EventProps } from '../../utils/common';
 import { generateTestDataId } from '../../utils/helpers';
 import { AcceptedColorComponentTypes } from '../../utils/themeFunctions';
 import { TestId } from '../../utils/types';
-import { buttonSpanStyle, buttonStyle, childrenWrapperStyle, iconStyle } from './Button.style';
+import {
+  buttonSpanStyle,
+  buttonStyle,
+  centralizedLoader,
+  childrenWrapperStyle,
+  iconStyle,
+} from './Button.style';
 import { useTypeColorToColorMatch } from '../../hooks/useTypeColorToColorMatch';
+import { useLoading } from 'hooks/useLoading';
+import Loader from '../Loader';
+import { useRef } from 'react';
```

Comment:
there is already everything imported from `react` in line 4
`import * as React from 'react';`. How about to do the below?
+ either `import * as React from 'react';` and `React. useRef<HTMLSpanElement>(null)`, or
+ `import React, { useRef } from 'react';`